### PR TITLE
fix(ga): fix ga resource lint error

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
fix ga resource lint error

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
./scripts/codecheck.sh ./huaweicloud/services/ga

==> Checking for running environment...

==> Applying patch...

==> Checking for code complexity...
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Language                              Files     Lines   Blanks  Comments     Code Complexity Complexity/Lines
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Go                                        5      2888      323        56     2509        325            65.06
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
~ces/ga/resource_huaweicloud_ga_listener.go       615       67        11      537         72            13.41
~/ga/resource_huaweicloud_ga_accelerator.go       666       72        11      583         67            11.49
~/resource_huaweicloud_ga_endpoint_group.go       557       63        11      483         66            13.66
~ces/ga/resource_huaweicloud_ga_endpoint.go       525       62        12      451         61            13.53
~ga/resource_huaweicloud_ga_health_check.go       525       59        11      455         59            12.97
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Total                                     5      2888      323        56     2509        325            65.06
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Processed 98659 bytes, 0.099 megabytes (SI)
─────────────────────────────────────────────────────────────────────────────────────────────────────────────

the TOP10 most complex functions:
7 ga deleteListenerWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_listener.go:554:1
7 ga updateListenerWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_listener.go:453:1
7 ga createListenerWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_listener.go:235:1
7 ga deleteHealthCheckWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_health_check.go:463:1
7 ga updateHealthCheckWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_health_check.go:361:1
7 ga createHealthCheckWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_health_check.go:180:1
7 ga deleteEndpointGroupWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_endpoint_group.go:495:1
7 ga updateEndpointGroupWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_endpoint_group.go:393:1
7 ga createEndpointGroupWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_endpoint_group.go:206:1
7 ga deleteEndpointWaitingForStateCompleted huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go:450:1
Average: 3.75

==> Checking for golangci-lint...

==> Checking for Nolint directives...

==> Checking for TF features in ga...

==> Checking for misspell in ga...

==> Checking for code complexity in ./huaweicloud/services/acceptance/ga...
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Language                              Files     Lines   Blanks  Comments     Code Complexity Complexity/Lines
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Go                                        5       676       67         5      604         25            20.23
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
~a/resource_huaweicloud_ga_endpoint_test.go       160       15         1      144          9             6.25
~urce_huaweicloud_ga_endpoint_group_test.go       121       13         1      107          4             3.74
~source_huaweicloud_ga_health_check_test.go       126       13         1      112          4             3.57
~esource_huaweicloud_ga_accelerator_test.go       126       13         1      112          4             3.57
~a/resource_huaweicloud_ga_listener_test.go       143       13         1      129          4             3.10
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Total                                     5       676       67         5      604         25            20.23
─────────────────────────────────────────────────────────────────────────────────────────────────────────────
Processed 19604 bytes, 0.020 megabytes (SI)
─────────────────────────────────────────────────────────────────────────────────────────────────────────────

the TOP5 most complex functions:
4 ga testEndpointImportState huaweicloud/services/acceptance/ga/resource_huaweicloud_ga_endpoint_test.go:144:1
3 ga getListenerResourceFunc huaweicloud/services/acceptance/ga/resource_huaweicloud_ga_listener_test.go:18:1
3 ga getHealthCheckResourceFunc huaweicloud/services/acceptance/ga/resource_huaweicloud_ga_health_check_test.go:18:1
3 ga getEndpointResourceFunc huaweicloud/services/acceptance/ga/resource_huaweicloud_ga_endpoint_test.go:18:1
3 ga getEndpointGroupResourceFunc huaweicloud/services/acceptance/ga/resource_huaweicloud_ga_endpoint_group_test.go:18:1
Average: 1.62

==> Checking for golangci-lint in ./huaweicloud/services/acceptance/ga...

==> Checking for Nolint directives in ./huaweicloud/services/acceptance/ga...

==> Cleanup patch...

Check Completed!

```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/ga"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ga -v  -timeout 360m -parallel 4
=== RUN   TestAccAccelerator_basic
=== PAUSE TestAccAccelerator_basic
=== RUN   TestAccEndpointGroup_basic
=== PAUSE TestAccEndpointGroup_basic
=== RUN   TestAccEndpoint_basic
=== PAUSE TestAccEndpoint_basic
=== RUN   TestAccHealthCheck_basic
=== PAUSE TestAccHealthCheck_basic
=== RUN   TestAccListener_basic
=== PAUSE TestAccListener_basic
=== CONT  TestAccAccelerator_basic
=== CONT  TestAccHealthCheck_basic
=== CONT  TestAccEndpoint_basic
=== CONT  TestAccEndpointGroup_basic
--- PASS: TestAccAccelerator_basic (62.66s)
=== CONT  TestAccListener_basic
--- PASS: TestAccEndpointGroup_basic (134.66s)
--- PASS: TestAccListener_basic (96.83s)
--- PASS: TestAccHealthCheck_basic (160.87s)
--- PASS: TestAccEndpoint_basic (186.04s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ga        186.083s

```

### CODE REVIEW

File: huaweicloud/services/ga/resource_huaweicloud_ga_accelerator.go [hunk 1]
```diff
@@ -484,7 +484,7 @@ func resourceAcceleratorUpdate(ctx context.Context, d *schema.ResourceData, meta
 	return resourceAcceleratorRead(ctx, d, meta)
 }
 
-func buildUpdateAcceleratorBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildUpdateAcceleratorBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```
Comment (ShiChangkuo) [inline huaweicloud/services/ga/resource_huaweicloud_ga_accelerator.go]: it's better to remove the second parameter

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go [hunk 2]
```diff
@@ -159,7 +159,7 @@ func resourceEndpointCreate(ctx context.Context, d *schema.ResourceData, meta in
 	return resourceEndpointRead(ctx, d, meta)
 }
 
-func buildCreateEndpointBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildCreateEndpointBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```
Comment (ShiChangkuo) [inline huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go]: ditto

File: huaweicloud/services/ga/resource_huaweicloud_ga_accelerator.go (hunk 1 above)
Comment (profoundwu) [reply huaweicloud/services/ga/resource_huaweicloud_ga_accelerator.go]: fixed

File: huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go (hunk 2 above)
Comment (profoundwu) [reply huaweicloud/services/ga/resource_huaweicloud_ga_endpoint.go]: fixed

**Pull request merged**